src/plone/base/interfaces/recyclebin.py

```diff
@@ -0,0 +1,127 @@
+"""Interfaces for the Plone Recycle Bin functionality."""
+
+from zope import schema
+from zope.interface import Interface
+
+
+class IRecycleBinControlPanelSettings(Interface):
+    """Interface for recycle bin settings"""
+
+    recycling_enabled = schema.Bool(
+        title="Enable the recycle bin",
+        description="Enable or disable the recycle bin feature.",
```

Comment:
@rohnsha0 We need to mark all the user-facing strings as translatable.

To do this, import the message factory (`from plone.base import PloneMessageFactory as _`) and set values like this: `title=_("Enable the recycle bin")`

See https://github.com/plone/plone.base/blob/main/src/plone/base/interfaces/controlpanel.py for examples

Reply:
done in https://github.com/plone/plone.base/pull/81/commits/6550420e157c5e57f27d2409071b19fc54cc8de3